src/api/bookstoreApi.js

```diff
@@ -0,0 +1,60 @@
+import { loadBooks } from '../redux/books/Books';
+
+const api = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi';
+const appId = 'A5M1SMeOFloqjitxesLY';
+
+const createApp = async () => {
+  const url = `${api}/apps/`;
+  const response = await fetch(url, {
+    method: 'POST',
+  });
+  const data = await response.text();
+  // Needed number to create app
+  console.log(data); // eslint-disable-line
```

Comment:
- [ ] Please remove all  `console.log()`  from your codebase it exposes important data to the user browser console